src/common/options.cc

```diff
@@ -776,6 +776,10 @@ std::vector<Option> get_global_options() {
     .set_default(5)
     .set_description(""),
 
+    Option("qat_compressor_enabled", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
+    .set_default(false)
+    .set_description(""),
```

Comment:
might want to document this option.

Reply:
Ok.

Reply:
@qwren the comment will not be visible to enduser. that's why i suggest add the description here.

Reply:
@qwren it's not addressed.

Reply:
Ok. will do.